CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Main (unreleased)
 ### Breaking changes
 
 - Set `retry_on_http_429` to `true` by default in the `queue_config` block in static mode's `remote_write`. (@wildum)
+- Renamed `non-indexed-labels` Loki processing stage to `structured_metadata`. (@vlad-diachenko)
```

Comment:
Breaking changes should also have an entry in relevant release notes:

- Static mode: https://github.com/grafana/agent/blob/main/docs/sources/static/release-notes.md
- Flow mode: https://github.com/grafana/agent/blob/main/docs/sources/flow/release-notes.md

Can you follow the existing pattern and add an entry matching this change under a new v0.37 header?

Reply:
done ;) thanks